modules/virt-creating-vm-snapshot-cli.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+[id="virt-creating-vm-snapshot-cli_{context}"]
+= Creating an virtual machine snapshot in the CLI
+
+You can create a virtual machine (VM) snapshot for an offline or online VM by creating a `VirtualMachineSnapshot` object. Kubevirt will coordinate with the QEMU guest agent to create a snapshot of the online VM.
+
+[NOTE]
+====
+In order to create snapshots of an online (Running state) VM with the highest integrity, we recommend you install the QEMU guest agent.
+
+The QEMU guest agent ensures a consistent snapshot by attempting to quiesce the VM's filesystem. This insures that in-flight I/O is written to the disk before the snapshot is taken and ensures that VMs with multiple disks have a coherent snapshot. If the guest agent is not present, quiescing is not possible and a best effort snapshot is taken. The conditions under which the snapshot was taken are reflected in the snapshot indications that are displayed in the web console or CLI.
+====
+
+.Prerequisites
+
+* Ensure that the persistent volume claims (PVCs) are in a storage class that supports Container Storage Interface (CSI) volume snapshots.
+* Install the OpenShift CLI (`oc`).
+* Power down the VM for which you want to create a snapshot.
```

Comment:
This step is optional.

Reply:
Good catch:

* Optionally, power down the VM for which you want to create a snapshot.